services/frontend/pkg/revaconfig/config.go

```diff
@@ -88,6 +89,17 @@ func FrontendConfigFromStruct(cfg *config.Config) map[string]interface{} {
 					"transfer_shared_secret": cfg.TransferSecret,
 					"timeout":                86400,
 					"insecure":               cfg.AppHandler.Insecure,
+					"webbaseuri":             "https://ocis.owncloud.test/external",
+					"web": map[string]interface{}{
+						"urlparamsmapping": map[string]string{
+							// param -> value mapper
+							"fileId": "fileid",
+							"app":    "appname",
+						},
+						"staticurlparams": map[string]string{
+							"contextRouteName": "files-spaces-personal",
```

Comment:
We can drop the `contextRouteName` from the params when https://github.com/owncloud/web/pull/7437 is merged, released and embedded into ocis.